custom_components/remote_homeassistant/__init__.py

```diff
@@ -47,6 +54,7 @@
 
 INSTANCES_SCHEMA = vol.Schema({
     vol.Required(CONF_HOST): cv.string,
+    vol.Required(CONF_STATE_ENTITY_NAME): cv.string,
```

Comment:
please don't add a new required attribute, as this adds a breaking change
The name of the sensor could be automatcally generated, like `sensor.remote_connection_<host>_<port>`

Reply:
changed, please see new commit :)